pkg/sdk/go/channels_test.go

```diff
@@ -386,7 +386,7 @@ func TestChannelsByThing(t *testing.T) {
 			desc:     "get a list of channels by thing with limit greater than max",
 			thing:    tid,
 			token:    token,
-			offset:   0,
+			offset:   Offset,
```

Comment:
This can be probably defined without to export, i.e `offset` and `limit`

Reply:
I'd define consts per test, no need to export. Eventually use different values for each test.

Reply:
Done